### PR TITLE
refactor(mint): remove unused context parameter from getProportions

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #XXX

## What is the purpose of the change

The context parameter in `getProportions` is unused. Removed the unused parameter.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? no
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? no
  - How is the feature or change documented? not applicable

### CODE REVIEW

File: x/mint/keeper/hooks_test.go [hunk 1]
```diff
@@ -50,8 +50,8 @@ func (suite *KeeperTestSuite) TestEndOfEpochMintedCoinDistribution() {
 
 		mintParams = app.MintKeeper.GetParams(ctx)
 		mintedCoin := app.MintKeeper.GetMinter(ctx).EpochProvision(mintParams)
-		expectedRewardsCoin, err := keeper.GetProportions(ctx, mintedCoin, mintParams.DistributionProportions.Staking)
-		suite.Require().NoError(err)
```
Comment (AlpinYukseloglu) [inline x/mint/keeper/hooks_test.go:54]: Out of curiosity, what was the reasoning behind removing `Require().` here?

**Review by AlpinYukseloglu — APPROVED** [review]
LGTM, left a minor question

File: x/mint/keeper/hooks_test.go (hunk 1 above)
Comment (stackman27) [reply x/mint/keeper/hooks_test.go:54]: I remember this being discussed [here](https://github.com/osmosis-labs/osmosis/pull/2046#pullrequestreview-1041071475), but i though the conclusion was to keep the `Require()`

File: x/mint/keeper/hooks_test.go [hunk 2]
```diff
@@ -50,8 +50,8 @@ func (suite *KeeperTestSuite) TestEndOfEpochMintedCoinDistribution() {
...
+		expectedRewardsCoin, err := keeper.GetProportions(mintedCoin, mintParams.DistributionProportions.Staking)
+		suite.NoError(err)
```
Comment (p0mvn) [inline x/mint/keeper/hooks_test.go:54]:
```suggestion
		suite.Require().NoError(err)
```

File: x/mint/keeper/hooks_test.go (hunk 1 above)
Comment (p0mvn) [reply x/mint/keeper/hooks_test.go:54]: merge conflict - fixed :)

**Pull request merged**